docs/data/sdks/typescript-react-native/migration.md

```diff
@@ -3,6 +3,326 @@ title: React Native SDK Migration Guide
 description: Use this guide to easily migrate from Amplitude's maintenance React Native SDK (@amplitude/react-native) to the new SDK (@amplitude/analytics-react-native).
 ---
 
+Amplitude New React Native SDK (`@amplitude/analytics-react-native`) features a plugin architecture, built-in type definition and broader platform support. New React Native SDK isn't backwards compatible with maintenance React Native SDK `@amplitude/react-native`. 
```

Comment:
Can we also mention that the unsent events or the identity-related value will be dropped and the effect?

Reply:
>  unsent events or the identity-related value will be dropped

Could you elaborate on this?

Reply:
previously we stored the deice id, user id, might be also session id and others in sqlite db. Since we don't have any migration logic, that stuff won't be migrated. That means, the deviceId might be regenerated while using new rn sdk.

Reply:
Add an admonition at the top of the page:

The Maintenance React Native SDK stores device ID, user ID, etc., in SQLite. However, there is no migration logic available, so this data won't be automatically migrated when upgrading to the new React Native SDK. Consequently, for example, device ID may be regenerated when using the new SDK.